jobscheduler/scheduler-test-component-shared/src/java/org/sakaiproject/component/app/scheduler/jobs/TestConfigurableJob.java

```diff
@@ -76,15 +76,15 @@ private final void readStringProperty()
 
         if (temp == null)
         {
-            log.debug ("string property is null");
+            log.info ("string property is null");
         }
         else if (temp.trim().length() == 0)
         {
-            log.debug ("string property is empty");
+            log.info ("string property is empty");
         }
         else
         {
-            log.debug ("string property is set to: '" + temp + "'");
+            log.info ("string property is set to: '" + temp + "'");
```

Comment:
log.info("string property is set to: '{}'", temp); (no big deal)

Reply:
Yeah, I can update this. I wasn't changing too much here but I thought it would be nicer if this job printed these values out since they'd be really just running in a test mode and might be hard on a nightly server to change the log level.